easybuild/easyblocks/s/suitesparse.py

```diff
@@ -94,8 +94,12 @@ def configure_step(self):
         try:
             for line in fileinput.input(fp, inplace=1, backup='.orig'):
                 for (k, v) in cfgvars.items():
-                    line = re.sub(r"^(%s\s*=\s*).*$" % k, r"\1 %s # patched by EasyBuild" % v, line)
-                    if k in line:
+                    orig_line = line
+                    line = re.sub(r"^\s*(%s\s*=\s*).*$" % k,
+                                  r"\1 %s # patched by EasyBuild" % v,
+                                  line)
```

Comment:
When doing a substitution, I find it useful to be able to visually compare the two expressions, aligning them vertically helps avoiding bugs.